src/main/webapp/assets/modules/common/model/challengeRegistrant.js

```diff
@@ -56,7 +56,7 @@ techlooper.filter("challengeRegistrant", function (apiService, $rootScope, jsonV
       }
       //delete registrant.$savedCriteria;
 
-      _.each(registrant.criteria, function (cri) {(!_.isNumber(cri.score)) && (cri.score = 0);});
+      _.each(registrant.criteria, function (cri) {(!$.isNumeric(cri.score)) && (cri.score = 0);});
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.